logstash-input-beats.gemspec

```diff
@@ -22,7 +22,7 @@ Gem::Specification.new do |s|
   s.add_runtime_dependency "logstash-core-plugin-api", "~> 1.0"
 
   s.add_runtime_dependency "logstash-codec-plain"
-  s.add_runtime_dependency "concurrent-ruby", "~> 0.9.2"
+  s.add_runtime_dependency "concurrent-ruby", "~> 1.0.0"
```

Comment:
Nothing justify to only use ~> 1.0.0, This plugins doesn't use any new feature that are only available in 1.0.0.
If we switch the dependency to 1.0.0 in the plugin, we will have to backport the plugin to 2.0 since 2.X is still stuck on 0.9.2.

I believe we can use this dependency, so It will work on either 0.9.X and 1.0.X
```
 s.add_runtime_dependency "concurrent-ruby", "~> 0.9.2", "~> 1.0.0"
```